### PR TITLE
Add explanation of the Presto Kerberos + SSL + snowflake problem

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by potiuk** [comment]
cc: @eladkal

**Comment by potiuk** [comment]
This will automatically make it into generated README,.mf when provider/backport provider packages are generated.

**Review by eladkal — APPROVED** [review]

File: airflow/providers/presto/ADDITIONAL_INFO.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+## Additional limitations
+
+The presto provider might not work correctly with Kerberos + SSL when it is installed together with the
+Snowflake provider. It's because of Snowflake monkeypatching the urllib3
+library as described in [this issue](https://github.com/snowflakedb/snowflake-connector-python/issues/324)
```
Comment (XD-DENG) [inline airflow/providers/presto/ADDITIONAL_INFO.md:24]:
```suggestion
library as described in [this issue](https://github.com/snowflakedb/snowflake-connector-python/issues/324).
```

File: airflow/providers/presto/ADDITIONAL_INFO.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+the offending code is [here](https://github.com/snowflakedb/snowflake-connector-python/blob/133d6215f7920d304c5f2d466bae38127c1b836d/src/snowflake/connector/network.py#L89-L92)
```
Comment (XD-DENG) [inline airflow/providers/presto/ADDITIONAL_INFO.md:25]:
```suggestion
The offending code is [here](https://github.com/snowflakedb/snowflake-connector-python/blob/133d6215f7920d304c5f2d466bae38127c1b836d/src/snowflake/connector/network.py#L89-L92)
```

**Review by XD-DENG — APPROVED** [review]
Minor change to suggest.

**Pull request merged**